File: src/main/python/ExposedStateMachine.java
#!/usr/bin/env python
""" generated source for module ExposedStateMachine """
# package: nl.insomnia247.nailbiter.eldrinkopubbot.state_machine
class ExposedStateMachine(StateMachine, InputMessage, OutputMessage):
    """ generated source for class ExposedStateMachine """
    def __init__(self, state):
        """ generated source for method __init__ """
        super(ExposedStateMachine, self).__init__(state)

    def setState(self, state):
        """ generated source for method setState """
        _setState(state)

    def getState(self):
        """ generated source for method getState """
        return _currentState

File: py/state_machine/exposed_state_machine.py
"""===============================================================================

        FILE: ./py/state_machine/exposed_state_machine.py

       USAGE: (not intended to be directly executed)

 DESCRIPTION: 

     OPTIONS: ---
REQUIREMENTS: ---
        BUGS: ---
       NOTES: ---
      AUTHOR: Alex Leontiev (alozz1991@gmail.com)
ORGANIZATION: 
     VERSION: ---
     CREATED: 2021-02-02T22:08:09.151139
    REVISION: ---

==============================================================================="""

from py.state_machine.state_machine import StateMachine


class ExposedStateMachine(StateMachine):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

    def setState(self, state):
        self._setState(state)

    def getState(self):
        return self._current_state
